monorepo/website/src/pages/docs/how-to/revoke-sequences.mdx

```diff
@@ -10,7 +10,7 @@ You can read more about the circumstances under which sequences could/should be
 Note that revoking a sequence **does not delete it**.
 While it will not turn up in default searches, and will be clearly marked as revoked, it will still be publicly visible.
 
-Deletion is generally _not_ permitted by Pathoplexus, since this is intransparent and causes confusion when a record cannot be traced.
+Deletion is generally _not_ permitted by Pathoplexus, since this is non-transparent and causes confusion when a record cannot be traced.
```

Comment:
```suggestion
Deletion is generally _not_ permitted by Pathoplexus, since this is intransparent and causes confusion when a record cannot be traced.
```